packages/core/src/components/ApplicationLauncher/Wrappers/ApplicationLauncherWrapper.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ApplicationLauncher from '../ApplicationLauncher';
+import ApplicationLauncherItem from '../ApplicationLauncherItem';
+
+const ApplicationLauncherWrapper = ({
+  apps,
+  noIcons,
+  grid,
+  tooltipPlacement
+}) => {
+  const renderApps = apps.map((app, i) => (
+    <ApplicationLauncherItem
+      key={i}
+      icon={app.icon}
+      title={app.title}
+      tooltip={app.tooltip}
+      tooltipPlacement={tooltipPlacement}
+      onClick={app.onClick}
+      noIcons={noIcons}
+    />
+  ));
+
+  return (
+    <ApplicationLauncher grid={grid} tooltipPlacement={tooltipPlacement}>
+      {renderApps}
+    </ApplicationLauncher>
+  );
+};
+ApplicationLauncherWrapper.propTypes = {
+  /** Array of App Objects */
+  apps: PropTypes.array,
+  /** No Icons Bool */
+  noIcons: PropTypes.bool,
+  /** Grid instead of List (List is Default) */
+  grid: PropTypes.bool,
+  /** Tooltip Placement */
+  tooltipPlacement: PropTypes.string
```

Comment:
Should take a tooltip to pass to the ApplicationLauncher to pass to the ApplicationLauncherToggle